apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QRowGetIT.java

```diff
@@ -847,7 +847,9 @@ public void getRowsWithTimestampQuery() {
             map("id", 3, "firstName", "Graham", "created", "2021-04-22T18:42:22.139-0800"),
             map("id", 4, "firstName", "Joel", "created", "2021-04-22T18:42:22.139+07"),
             map("id", 4, "firstName", "Deborah", "created", "2021-04-22T18:42:22.139-05"),
-            map("id", 5, "firstName", "Timothy", "created", timestampAsMsecs)));
+            map("id", 5, "firstName", "Timothy", "created", timestampAsMsecs),
+            // 13-Jul-2023, tatu: [stargaate#2674] Support microsecond fractions too
+            map("id", 6, "firstName", "Jerome", "created", "2023-07-13T14:02:06.018850Z")));
```

Comment:
Can we also read this back and check if it's retaining only up to milliseconds precision and dropping the microseconds part?
Also, do we need to document this somewhere that the microseconds fraction will be allowed but dropped?

Reply:
Dropping of value is verified by the unit test (`ToProtoConverterTest.java`) -- value compared to has only milliseconds.

Documentation is a good question, need to work with @johnsmartco on how to do that (in context where date/time values, allowed are discussed). I think it is related to Cassandra only storing millisecond timestamp.